base_controller_lib/build.gradle

```diff
@@ -14,17 +14,26 @@
  * the License.
  */
 
+// UsbSerialLibrary is not yet available on Maven, so the path to the artifact
+// has to be sourced to work properly with tools like AndroidStudio.
 dependencies {
-    compile 'org.ros.android_core:android_10:[0.2, 0.3)'
-    compile 'com.hoho.android.usbserial:UsbSerialLibrary:[0.1.1, 0.2)'
+    compile 'org.ros.rosjava_core:rosjava:[0.3.0]'
```

Comment:
it is recommended to keep an open format `[0.3, 0.4)` so that this will always get the latest 0.3 version. In this way, it will always only use 0.3.0, even if 0.3.1 becomes available. Was that your intention?